roles/validate_operator_bundle/tasks/main.yml

```diff
@@ -28,10 +28,49 @@
     bundle_validate: "{{ operator_work_dir }}"
   when: not run_upstream|bool
 
-- name: "Set the operatorhub checks if enabled"
+- name: "list work_dir"
+  shell: "ls {{ work_dir }}"
+
+- name: "Get supported OCP versions"
+  block:
+    - name: Get skopeo inspect json
+      shell: "cat {{ work_dir }}/bundle-skopeo-inspect.json"
+      register: bundle_skopeo_inspect_json
+
+    - name: Convert json to map
+      set_fact:
+        bundle_skopeo_inspect_map: "{{ bundle_skopeo_inspect_json.stdout | from_json }}"
+
+    - name: Determine ocp version annotation
+      set_fact:
+        ocp_version_annotation: "{{ bundle_skopeo_inspect_map['Labels']['com.redhat.openshift.versions'] }}"
+
+    - name: "Query Pyxis"
+      uri:
+        url: "https://catalog.redhat.com/api/containers/v1/operators/indices?sort_by=data.ocp_version&ocp_versions_range={{ ocp_version_annotation }}&organization=redhat-operators"
+        method: GET
+        headers:
+          Accept: "application/json"
+        status_code: 200
+        return_content: true
+      register: supported_indices_json
+
+    - name: "Convert output to map"
+      set_fact:
+        supported_indices: "{{ supported_indices_json.content | from_json }}"
+
+    - name: "Parse values returned from Pyxis"
+      set_fact:
+        highest_index: "{{ supported_indices['data'] | last }}"
+
+    - name: "Determine if v4.9 is supported"
+      set_fact:
+        support_v4_9: "{{ highest_index['ocp_version'] is version('4.9', '>=') }}"
```

Comment:
Code freeze for OCP v4.9 already happen so we can use stricter check
and `support_v4_9` should be true when v4.9 is in the range.

older version was weaker "only v4.9" was supported

Reply:
This check covers the stricter version.  It lists all of the supported OCP versions and if the highest version is 4.9 or greater, it adds the check